.github/workflows/update-checksums.yml

```diff
@@ -44,217 +48,27 @@ jobs:
-              const pullRequestPartialRef = `heads/${process.env.PR_BRANCH}`;
-              const pullRequestFullRef = `refs/${pullRequestPartialRef}`;
-
-              core.info(`Creating new tree:
-                owner:     ${context.repo.owner}
-                repo:      ${context.repo.repo}
-                base_tree: ${githubSHA}
-              `);
-
-              // read the file content
-              const checksumFilePath = `${githubWorkspace}/terraform-checksums.json`;
-              const content = await fs.readFile(checksumFilePath, { encoding: "utf8" });
-
-              // create new git tree from the pr branch
-              const { data: tree } = await github.rest.git.createTree({
-                owner: context.repo.owner,
-                repo: context.repo.repo,
-                base_tree: githubSHA,
-                tree: [
-                  {
-                    path: "terraform-checksums.json",
-                    mode: "100644",
-                    type: "blob",
-                    content: content,
-                  },
-                ],
-              });
-
-              core.debug("tree: ", tree);
-
-              core.info(`Creating new commit:
-                owner:   ${context.repo.owner}
-                repo:    ${context.repo.repo}
-                parents: ${parentSHA}
-                tree:    ${tree.sha}
-              `);
-
-              // create a commit from on the git tree
-              const { data: commit } = await github.rest.git.createCommit({
-                owner: context.repo.owner,
-                repo: context.repo.repo,
-                message: "chore: [automated] checksum updates",
-                parents: [parentSHA],
-                tree: tree.sha,
-              });
-
-              core.debug("commit: ", commit);
-
-              core.info(`Updating PR branch ref
-                owner: ${context.repo.owner}
-                repo:  ${context.repo.repo}
-                ref:   ${pullRequestPartialRef}
-                sha:   ${commit.sha}
-              `);
-
-              // update the pr branch reference with the new git tree
-              await github.rest.git.updateRef({
-                owner: context.repo.owner,
-                repo: context.repo.repo,
-                ref: pullRequestPartialRef,
-                sha: commit.sha,
-              });
-            } catch (err) {
-              core.error(err);
-              core.setFailed(`Failed to create commits for pull request branch: ${err}`);
             }
 
+      # Create a pull request with updated files
       - name: 'Create/Update Pull Request'
-        id: 'create-update-pull-request'
         if: '${{ env.CHANGES }}'
-        uses: 'actions/github-script@98814c53be79b1d30f795b907e553d8679345975' # ratchet:actions/github-script@v6
+        uses: 'abcxyz/pkg/.github/actions/create-pull-request@cbbfe9eb1d44a61f1743f8e25e1b0d0fda7f0677' # ratchet:abcxyz/pkg/.github/actions/create-pull-request@main
```

Comment:
I think we should `ratchet:exclude` the stuff we own.

Reply:
done